pagopa/pn-frontend
# Replaced translations for landing app, downgraded MUI Italia package

## Short description
Replaced translations for landing app.

## List of changes proposed in this pull request
- Replaced translations for SL, EN, FR and DE languages in landing app
- @pagopa/mui-italia package has been downgraded from 0.9.0 to 0.8.12 as temporary fix for not launching app

## How to test
You need to reinstall packages in pn-landing before testing.
Then switch languages to see the new translations.
